scripts/make_running_note_from_pooling.py

```diff
@@ -42,20 +44,17 @@ def main(lims, args):
     now=datetime.now().strftime("%Y-%m-%d %H:%M:%S.%f")
     for pid in datamap:
         pj=Project(lims, id=pid)
-        running_notes=json.loads(pj.udf['Running Notes'])
         if len(datamap[pid]) > 1:
             rnt="{0} samples planned for {1}".format(len(datamap[pid]), wsname)
         else:
             rnt="{0} sample planned for {1}".format(len(datamap[pid]), wsname)
 
-        running_notes[now]={"note": rnt, "user" : username, "email":user_email, "category":"Workset"}
-
-        pj.udf['Running Notes']=json.dumps(running_notes)
-        pj.put()
+        running_note = {"note": rnt, "user" : username, "email":user_email, "category":"Workset"}
```

Comment:
I wouldn't mind a even more of a whitespace cleanup here. Removing whitespace after `"user"` and adding a space after `"category:"`

Reply:
Done!